modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -19,15 +19,16 @@ def get_fault_info
       @fault_code = fault&.dig('faultcode')&.split(':')&.dig(1)
       @fault_string = fault&.dig('faultstring')
       @fault_message = fault&.dig('detail', 'MessageException') || fault&.dig('detail', 'MessageFaultException')
-      return {} if @fault_string.include?('IntentToFileWebService') && @fault_string.include?('not found')
 
       get_exception
     end
 
     private
 
     def get_exception
-      if not_found?
+      if veteran_not_found?
+        { status: '404', detail: 'Veteran ID not found' }
```

Comment:
Return custom object to be handled in our controller for this specific scenario so we don't return 404 on ALL empty responses from BGS.

Reply:
Also has to be ahead of the existing "not_found" below in order to not get caught by it